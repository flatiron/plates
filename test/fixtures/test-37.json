{
  "author": [
    { "name": {"inner": "Author 1"}, "title": "Title 1", "link": "//1" },
    { "name": {"inner": "Author 2"}, "title": "Title 2", "link": "//2" },
    { "name": {"inner": "Author 3"}, "title": "Title 3", "link": "//3" }
  ]
}