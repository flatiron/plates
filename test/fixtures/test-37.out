<div class="authors">
  <ul>
    <li class="author">
      <h2 class="title">Title 1</h2>
      <a class="name" href="//1">The author name is<span class="inner">Author 1</span></a>
    </li><li class="author">
      <h2 class="title">Title 2</h2>
      <a class="name" href="//2">The author name is<span class="inner">Author 2</span></a>
    </li><li class="author">
      <h2 class="title">Title 3</h2>
      <a class="name" href="//3">The author name is<span class="inner">Author 3</span></a>
    </li>
  </ul>
</div>